Feature/ONG4-32/Edición de home

gabrielpindo opened the pull request:
Se creó un componente <EditHome /> que edita los sliders y el texto de bienvenida del área de inicio de la página para la ONG "Somos más".

gabrielpindo committed:
Update .eslintrc.js

gabrielpindo committed:
Edición de Inicio, formulario

gabrielpindo committed:
Más modificaciones para aplicar los cambios al welcome_text

gabrielpindo committed:
.

gabrielpindo committed:
.

gabrielpindo committed:
Update de edicionHome Ultimos cambios

gabrielpindo committed:
Solucionado lo de la modificación del Welcome_Text de <EditHome />

gabrielpindo committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-32/EdicionHome

gabrielpindo committed:
update editHome

DiegoSantelices10 committed:
update

DiegoSantelices10 committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-32/EdicionHome

DiegoSantelices10 committed:
marge

DiegoSantelices10 committed:
update new

DiegoSantelices10 committed:
Validacion de imagen - update

gabrielpindo committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-32/EdicionHome

gabrielpindo committed:
Merge EditHome

milandre commented on src/Services/Organization/ApiService.js:
```diff
@@ -8,13 +8,23 @@ export const getOrganization = setData => {
+	}
+
+
+	export const putOrganization = (values, id) => {
```
Revisar esta linea que quedó mal indentada. ¿Van a dejar ambos endpoints (putOrganization, putOrganizationWelcomeText)? Tambien hay error de importación aca porque no esta el metodo getOrganizationWelcome

milandre reviewed (changes requested):
@gabrielpindo revisar esta rama nuevamente porque hay errores de importación, no está la ruta de /backoffice/home tampoco agregada al App.js

gabrielpindo committed:
Update de Formulario Home

gabrielpindo committed:
Correcciones del Formulario edición de home

Se decidió dejar solo la funcion putOrganization()

Se identó correctamente el componente

Se corrigió el error de importación del getOrganizationWelcome()

Se añadió al App.js la ruta /backoffice/home para acceder al componente

gabrielpindo committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-32/EdicionHome

gabrielpindo committed:
Update App.js

gabrielpindo committed:
Update rama

gabrielpindo committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-32/EdicionHome
